CHANGELOG.md

```diff
@@ -7,342 +7,298 @@
 
 ### [4.1.1](https://github.com/SelectQuoteLabs/SQForm/compare/v4.1.0...v4.1.1) (2021-03-09)
 
-
 ### Bug Fixes
 
-* 🐛 make only dialog body scroll ([60beb7b](https://github.com/SelectQuoteLabs/SQForm/commit/60beb7b1e1bcf4da4fd01a2debb6fd50310504ae)), closes [#17](https://github.com/SelectQuoteLabs/SQForm/issues/17)
+- 🐛 make only dialog body scroll ([60beb7b](https://github.com/SelectQuoteLabs/SQForm/commit/60beb7b1e1bcf4da4fd01a2debb6fd50310504ae)), closes [#17](https://github.com/SelectQuoteLabs/SQForm/issues/17)
```

Comment:
This file is auto-generated. @codyarose can you dig into what might be causing this issue? My guess is Prettier for VS Code is formatting this markdown file..

Reply:
Spoke with Cody and he's going to dig into this